features/features.go

```diff
@@ -523,6 +523,14 @@ var (
 						enableForClusterProfile(SelfManaged, configv1.DevPreviewNoUpgrade, configv1.TechPreviewNoUpgrade).
 						mustRegister()
 
+	FeatureGateNewOLMWebhookSupport = newFeatureGate("NewOLMOwnWebhookSupport").
+						reportProblemsToJiraComponent("olm").
+						contactPerson("pegoncal").
+						productScope(ocpSpecific).
+						enhancementPR("https://github.com/openshift/enhancements/pull/1774").
+						enableForClusterProfile(SelfManaged, configv1.DevPreviewNoUpgrade, configv1.TechPreviewNoUpgrade).
```

Comment:
This feature won't be available on HyperShift? Or just, not yet?

Reply:
afaik we don't have HyperShift slated anytime soon...but I believe eventually it will be the goal